src/neuroglancer/rendered_data_panel.ts

```diff
@@ -34,6 +34,9 @@ import {TouchEventBinder, TouchPinchInfo, TouchTranslateInfo} from 'neuroglancer
 import {getWheelZoomAmount} from 'neuroglancer/util/wheel_zoom';
 import {ViewerState} from 'neuroglancer/viewer_state';
 
+import './rendered_data_panel.css';
```

Comment:
Note: Some of these imports are duplicated now that I converted all of the require -> import in the master branch.

Reply:
Sorry, git merge was not my friend